content/beta/asset-manager/swagger/resources/asset_families/definitions/asset_family.yaml

```diff
@@ -54,12 +54,75 @@ properties:
                 type: string
               mode:
                 type: string
-
+  transformations:
+    type: array
+    description: The transformations to perform on source files in order to generate new files into your asset attributes. To understand the format of this property, see <a href='/beta/asset-manager/transformations-focus.html'>here</a>.
+    x-validation-rules: To know more about the validation of this property, take a look at our transformation format.
+    default: {}
+    items:
+      type: object
+      required: ['label', 'target', 'source', 'operations']
```

Comment:
is it possible to say "filename_suffix" or "filename_prefix" is needed ?

Reply:
Not with the Swagger format, unfortunately. You cannot say that you need at least one of them. But I said it inside the plain doc.